modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase

```diff
@@ -167,12 +165,12 @@ definition {
 	@description = "LPS-163243: The user can execute the upgrade:executeAll command on Gogo shell to execute all remaining upgrade processes"
 	@priority = 3
 	test ExecuteAllPendingUpgradesFromGogoShell {
+		property osgi.module.configuration.file.names = "com.liferay.portal.search.configuration.IndexStatusManagerConfiguration.config";
+		property osgi.module.configurations = "indexReadOnly=\"true\"";
```

Comment:
Disabled indexing during upgrade according to [LPP-47994](https://issues.liferay.com/browse/LPP-47994)

> Regarding this error, this seems like the customer did not set the Elasticsearch index to read-only. We recommend that customers set their Elasticsearch indices to read-only during the upgrade, because there are a large amount of changes that get made in the database during the upgrade, so Elasticsearch may not be able to handle indexing them all when they occur, and errors like this will result. After the upgrade completes, the client should then re-enable index-writing and do a full reindex. Please see [Step 8 of this article](https://help.liferay.com/hc/en-us/articles/360017896372-Preparing-an-Upgrade-to-Liferay-DXP-7-0#step-8-disable-indexing-during-the-upgrade-process) for more information about how to do that (note: the article was written for 7.0, but the information there in Step 8 should apply to 7.1 as well).

Reply:
@susanchen3 
This should be set automatically during the upgrade: https://github.com/liferay/liferay-portal/commit/8fa1895786654c2006a2b5537331582da850cf75

So this does not look right...